README.md

```diff
@@ -1,46 +1,65 @@
-# solid-client-authn
+# Solid JavaScript Authentication - solid-client-authn
 
-`solid-client-authn` is a set of libraries designed to authenticate with Solid identity servers,
-and thereafter to make authenticated requests to Solid storage servers from JavaScript clients
-deployed in any environment. The `solid-client-authn` libraries all have a common API
-, and come in different modules intended for different deployment environments:
+`solid-client-authn` is a set of libraries designed to authenticate with Solid identity servers. 
+The libraries share a common API and include different modules for different deployment environments:
 - `solid-client-authn-browser` can be used to help build web apps in the browser.
 - `solid-client-authn-node` is **planned** to help build server-side and console-based apps.
 
-## Related libraries
+@inrupt/solid-client-authn libraries are part of a family open source JavaScript libraries designed to support developers building Solid applications.
 
-### Read, write and transform data
+# Inrupt Solid JavaScript Client Libraries
 
-`solid-client-authn` libraries are all about authentication and fetching. To manipulate the data, have convenience methods to read and write to your Pod, manage access rights to your data, and more, you can have a look at [solid-client](https://www.npmjs.com/package/@inrupt/solid-client).
+## Data access and permissions management - solid-Client
 
-## Using the Appropriate API
+[@inrupt/solid-client](https://github.com/inrupt/solid-client-js) allows developers to access data and manage permissions on data stored in Solid Pods.
+
+## Authentication - solid-client-authn
+
+[@inrupt/solid-client-authn](https://github.com/inrupt/solid-client-authn) allows developers to authenticate against a Solid server. This is necessary when the resources on your Pod are not public.
```

Comment:
I wouldn't link back to the current README.

Reply:
I'd like to mention all the libraries in all the readmes for consistency.